hummingbird/ml/_topology.py

```diff
@@ -261,17 +265,15 @@ def fix_graph(graph):
             config["relay.FuseOps.max_depth"] = extra_config[constants.TVM_MAX_FUSE_DEPTH]
 
         # First we need to generate the torchscript model.
-        batch_trace_input, remainder_trace_input = _get_trace_input_from_test_input(
-            extra_config[constants.TEST_INPUT], batch_size
-        )
+        batch_trace_input, remainder_trace_input = _get_trace_input_from_test_input(test_input, remainder_size)
 
         tvm_model = _compile_tvm_model(topology, torch_model, batch_trace_input, target, ctx, config, extra_config)
 
         if remainder_trace_input is not None:
-            tvm_remainder_model = _compile_tvm_model(
+            remainder_model = _compile_tvm_model(
                 topology, torch_model, remainder_trace_input, target, ctx, config, extra_config
             )
-            extra_config[constants.TVM_REMAINDER_MODEL] = tvm_remainder_model
+            extra_config[constants.TVM_REMAINDER_MODEL] = remainder_model
```

Comment:
I don't think we need this anymore. Probably also `constants.TVM_REMAINDER_MODEL` can be removed.

Reply:
Good catch removed.